materials/introduction/02-gem5-basics/02-frequently-used-features/02-class-variables.py

```diff
@@ -0,0 +1,69 @@
+class Processor:
+    type_name = "Processor"
```

Comment:
I'd use something other than the type name as that could be confusing. Maybe `default_frequency = "3GHz"`?